File: Algorithms/2011_Final_Value_of_Variable_After_Performing_Operations/solution.go
func finalValueAfterOperations(operations []string) int {
    var result int
    result = 0
    
    for _, operand := range operations {
        if operand == "++X" || operand == "X++" {
            result++
        } else {
            result--
        }
    }
    
    return result
}

File: Algorithms/2011_Final_Value_of_Variable_After_Performing_Operations/solution.cpp
class Solution {
public:
    int finalValueAfterOperations(vector<string>& operations) {
        int result = 0;
        for (string operand :operations) {
            if (operand ==  "++X" || operand == "X++") {
                result++;
            } else {
                result--;
            }
        }
        return result;
    }
};